R/print.R

```diff
@@ -0,0 +1,87 @@
+#' Print method for `<multi_epidist>` class
+#'
+#' @param x A `<multi_epidist>` object.
+#' @param n A `numeric` specifying how many `<epidist>` objects to print. This
+#' argument is passed to [head()] for `list` printing. Default is `NULL` and the
+#' number of elements to print is controlled by package [options()].
+#' @inheritParams print.epidist
+#'
+#' @return Invisibly returns a `<multi_epidist>`. Called for side-effects.
```

Comment:
Could you add an example please?

Reply:
Added in 8c291aab3ed36a6ca55d6385532da27ee1e8b370.